docs/docs/cli/creating-environments.md

```diff
@@ -19,7 +19,7 @@ spec:
 In order to apply this configuration to your Tracetest instance, make sure to have your [CLI configured](./configuring-your-cli.md) and run:
 
 ```
-tracetest environment apply -f <environment.yaml>
+tracetest apply environment -f <environment.yaml>
```

Comment:
Is this page part of the docs sidebar? I don't see this page in our docs.

Reply:
It was removed by @adnanrahic 5 months ago. I'll ping him to know if that's intentional.

Reply:
I think we had an issue with the CLI at some point and I hid the page. I should have put it back in the sidebar eventually. Must have slipped through the cracks.